### DISCUSSION TITLE
le logiciel détecte tout et fais tout, mai ne répond pas quand je lui parle

### DISCUSSION THREAD

**Discussion by rafrourf** [Nash115/frenchATC-for-MSFS2020#38] [Help] [answered]
bonjour, j'ai un souci avec le logiciel, il détecte bien que je fai un vol vfr mais qd je démarre mon avion et veut lui parler, il ne me répond pas et aucune réaction dan le menu atc de flight simulator. Est-ce normal

**Comment by fp81** [comment, marked as answer]
Bonjour, j'ai eu le même pb et cela vient du fait que le site get-metar ne fonctionne plus. Du coup j'ai shunter les métro dans le code python et là tout fonctionne bien.

**Reply by rafrourf** [reply]
le problème a été réosolu, merci beaucoups
